tests/test_encrypt.py

```diff
@@ -0,0 +1,91 @@
+import os
+import pickle
+from io import BytesIO
+
+import pytest
+from sigal import init_plugins
+from sigal.gallery import Gallery
+from sigal.plugins.encrypt import endec
+from sigal.plugins.encrypt.encrypt import cache_key
+
+CURRENT_DIR = os.path.dirname(__file__)
+
+
+@pytest.fixture()
+def remove_cache(settings):
+    yield
+    cachepath = os.path.join(settings['destination'], ".encryptCache")
+    os.remove(cachepath)

```

Comment:
As the cache goes in a temp dir, why do you need to remove it ?

Reply:
I copied those lines from `tests/test_extended_caching.py` and thought it looked good... I didn't realize a temp dir already takes care of the cleaning.

Reply:
Removed in 2c4fdd6.

Reply:
Indeed, actually `test_extended_caching` should use tmpdir instead, I will fix this in another commit.